script/Arcueid/Day 6/06_00_ARC06_3_1.txt

```diff
@@ -1,88 +1,100 @@
 [f08d5e8b0b695b6f97adf84cb3c7d234fcfd195d]{
```

Comment:
this file shares a lot with Gekimon's PR for ciel. might want to coordinate there.